### DISCUSSION TITLE
external style sheets

### DISCUSSION THREAD

**Discussion by DuajDiaz** [DuajDiaz/Travel-Website#64] [Q&A] [answered]
Why are external style sheets preferred?

**Comment by Bunny77K9** [comment, marked as answer]
External style sheets provide an ample amount of advantages to developers. Some of the benefits are as follows:

    Classes can be reused any number of times.
    They allow for the style control of multiple documents through a single file.
    Selectors and grouping can be used to apply styles easily.
